gslib/addlhelp/dev.py

```diff
@@ -96,20 +96,19 @@
        ./gsutil test -u
 
      If you made changes to boto, please run the boto tests. For these tests you
-     need to use HMAC credentials (from gsutil config -a), because the current
-     boto test suite doesn't import the OAuth2 handler. You'll also need to
-     install some python modules. Change directories into the boto root
-     directory at third_party/boto and run:
+     need to use HMAC credentials (from gsutil config -a), because the boto test
+     suite doesn't import the OAuth2 handler. You'll also need to install some
+     python modules. Change directories into the boto root directory at
```

Comment:
Typo: `Python`

Reply:
Done, though I assumed the crux of the comment was capitalization and not code font. Let me know if I'm mistaken